homeassistant/components/airly/__init__.py

```diff
@@ -115,3 +126,13 @@ def __init__(self, session, api_key, latitude, longitude):
         except (ValueError, AirlyError, ClientConnectorError) as error:
             _LOGGER.error(error)
             self.data = {}
+
+        # Airly allows 100 requests per day. We check how many Airly config entries are
```

Comment:
This comment may easily go out of sync due to the `100` is defined wlsewhere

Reply:
Yes, you're right. I will change this.